.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto-convert line endings and set whitespace warnings for any (Git detected) text files
+* text=auto whitespace=trailing-space,tab-in-indent,space-before-tab,tabwidth=4
```

Comment:
This prevents us from having to ask every Windows developer to add a `core.autocrlf` setting to their local Git config.